chaco/ticks.py

```diff
@@ -242,6 +243,10 @@ def auto_ticks(
     An array of tick mark locations. The first and last tick entries are the
```

Comment:
nit - this is related but also tangential. the docstring says that this method returns an array - i am not sure if that means that we should strictly return an empty array.

this goes back to the general issue you mentioned of checking and enforcing return types across chaco. i think this can be an umbrella issue where we accumulate and fix issues that we discover.

Reply:
I saw this too, but then at the end of the method it appears to return just a list so an empty list is consistent.
Perhaps I should update the docstring instead?  Or maybe they both really should be arrays?

Reply:
I think I am going to push this through for now, but this can be re-evaluated as part of #662